### PR TITLE
Shorter Questionnaire + Error when appointment is already booked

### PR DESCRIPTION
**Pull request by yaron1m**

### CODE REVIEW

**Comment by yaron1m** [comment]
Shorter questions:
image


Error message:
image

File: packages/donor/src/screens/questionnaire/QuestionnaireScreen.tsx [hunk 1]
```diff
@@ -236,50 +126,20 @@ export default function QuestionnaireScreen({
           if (hasAlreadyDonated === "no") {
             setHasAlreadyDonated("");
           }
-          if (isWeightOver55 === "no") {
-            setIsWeightOver55("");
+          if (isWeightValid === "no") {
```
Comment (evbambly) [inline packages/donor/src/screens/questionnaire/QuestionnaireScreen.tsx:129]: Why are we comparing to a string and not bool or enum here?

File: packages/donor/src/screens/questionnaire/QuestionnaireScreen.tsx [hunk 2]
```diff
@@ -155,28 +76,17 @@ export default function QuestionnaireScreen({
...
       value={isRightAge}
       onChange={setIsRightAge}
-      label={"האם אתה בטווח הגילאים 17-65?"}
+      label={"האם הנך מעל גיל 17?"}
     />
   );
 
-  const [isPregnantEver, setIsPregnantEver] = useState("");
-  const IsPregnantEver = (
+  const [wasPregnant, setWasPregnantEver] = useState("");
```
Comment (evbambly) [inline packages/donor/src/screens/questionnaire/QuestionnaireScreen.tsx:83]: Why is this a string value?

**Review by evbambly — APPROVED** [review]

File: packages/donor/src/screens/questionnaire/QuestionnaireScreen.tsx (hunk 2 above)
Comment (yaron1m) [reply packages/donor/src/screens/questionnaire/QuestionnaireScreen.tsx:83]: I am not sure, but I didn't change this.
We can change this in a different PR

File: packages/donor/src/screens/questionnaire/QuestionnaireScreen.tsx (hunk 1 above)
Comment (yaron1m) [reply packages/donor/src/screens/questionnaire/QuestionnaireScreen.tsx:129]: I am not sure, but I didn't change this.
We can change this in a different PR

**Pull request merged**